qiita_pet/templates/study_description.html

```diff
@@ -222,6 +222,26 @@
   }
 }
 
+function delete_processed_data(processed_data_id) {
```

Comment:
Outside the scope of this pull request, but we really should start using the AJAX POST capabilities instead of this form creation/submission we're currently doing.